### PR TITLE
Bring Text Data up to SGPC parity

### LINKED ISSUE #1242: 403K & S86E
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/374/line/9
Source | Sri Guru Granth Sahib Ji
Page | 374
ID | S86E
Line | Awsw mhlw 5 pMcpdy ]<br>siB ibiD KoeI; loiB suAwn ]2]

(Look at 1st Line and the end of the 5th Line)
image

### LINKED ISSUE #1243: PQKA
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/471/line/5
Source | Sri Guru Granth Sahib Ji
Page | 471
ID | PQKA
Line | nw eyhu qutY. nw mlu lgY; nw eyhu jlY. n jwie ]

(Look at 5th Line)
image

### LINKED ISSUE #1244: SCQW
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/486/line/0
Source | Sri Guru Granth Sahib Ji
Page | 486
ID | SCQW
Line | rwm rswien; pIau ry dgrw ]3]4]

(See second line)
image

### LINKED ISSUE #1245: 9A9Z
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/588/line/10
Source | Sri Guru Granth Sahib Ji
Page | 588
ID | 9A9Z
Line | vrmI mwrI. swpu n mrY; iqau. ingury krm kmwih ]

(See 9th Line)
image

### LINKED ISSUE #1246: B5FS
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/608/line/5
Source | Sri Guru Granth Sahib Ji
Page | 608
ID | B5FS
Line | iks hau jwcI. iks AwrwDI; jw sBu ko kIqw hosI ]

(See 7th Line)
image

### LINKED ISSUE #1247: MVFY
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/703/line/31
Source | Sri Guru Granth Sahib Ji
Page | 703
ID | MVFY
Line | Xwr vy. ihk fUM ihk cwVY; hau iksu icqyhIAw ]

image

### LINKED ISSUE #1248: LR63
| Key | Value |
| --- | ----- |
View | https://database.shabados.com/sources/1/page/756/line/0
Source | Sri Guru Granth Sahib Ji
Page | 756
ID | LR63
Line | scw swhu scy vxjwry; EQY kUVy. n itkMin ]

(Second from last line)
image

### PR DESCRIPTION
**Pull request by Sarabveer**
Brings Sri Guru Granth Sahib Ji parity with SGPC printed Larivaar

Closes #1242 
Closes #1243
Closes #1244 
Closes #1245 
Closes #1246 
Closes #1247 
Closes #1248

### CODE REVIEW

File: data/Sri Guru Granth Sahib Ji/0210.json [hunk 1]
```diff
@@ -599,7 +599,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "rwgu gauVI cyqI mhlw 5 ]"
+          "SGPC": "rwgu gauVI cyqI₁ mhlw 5 ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0210.json]:
```suggestion
          "SGPC": "rwgu gauVI cy₁qI mhlw 5 ]"
```

File: data/Sri Guru Granth Sahib Ji/0214.json [hunk 2]
```diff
@@ -800,7 +800,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "rwgu gauVI mwlw mhlw 5 ]"
+          "SGPC": "rwgu gauVI mwlw₁ mhlw 5 ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0214.json]:
```suggestion
          "SGPC": "rwgu gauVI mw₁lw mhlw 5 ]"
```

File: data/Sri Guru Granth Sahib Ji/0215.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "gauVI mwlw mhlw 5 ]"
+          "SGPC": "gauVI mwlw₂ mhlw 5 ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0215.json]:
```suggestion
          "SGPC": "gauVI mw₂lw mhlw 5 ]"
```

File: data/Sri Guru Granth Sahib Ji/0216.json [hunk 4]
```diff
@@ -896,7 +896,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "rwgu gauVI mwJ mhlw 5 ]"
+          "SGPC": "rwgu gauVI mwJ₁ mhlw 5 ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0216.json]:
```suggestion
          "SGPC": "rwgu gauVI mw₁J mhlw 5 ]"
```

File: data/Sri Guru Granth Sahib Ji/0247.json [hunk 5]
```diff
@@ -14,7 +14,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "rwgu gauVI CMq mhlw 5 ]"
+          "SGPC": "rwgu gauVI₁ CMq mhlw 5 ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0247.json]:
```suggestion
          "SGPC": "rwgu g₁auVI CMq mhlw 5 ]"
```

Also note unicode displays this too far to the left

File: data/Sri Guru Granth Sahib Ji/0248.json [hunk 6]
```diff
@@ -14,7 +14,7 @@
         "pronunciation_information": "",
         "type": "Sirlekh",
         "gurmukhi": {
-          "SGPC": "gauVI mhlw 5 ]"
+          "SGPC": "gauVI₂ mhlw 5 ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0248.json]:
```suggestion
          "SGPC": "g₂auVI mhlw 5 ]"
```

File: data/Sri Guru Granth Sahib Ji/0332.json [hunk 7]
```diff
@@ -791,7 +791,7 @@
         "pronunciation_information": "",
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "rwgu gauVI bYrwgix kbIr jI ]"
+          "SGPC": "rwgu gauVI bYrwgix₁ kbIr jI ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/0332.json]:
```suggestion
          "SGPC": "rwgu gauVI bYrwg₁ix kbIr jI ]"
```

File: data/Sri Guru Granth Sahib Ji/1394.json [hunk 8]
```diff
@@ -800,7 +800,7 @@
         "pronunciation_information": "",
         "type": "Pankti",
         "gurmukhi": {
-          "SGPC": "iq nr dulIcY bhih; iq nr auQip ibQpih ]"
+          "SGPC": "iqnr dulIcY bhih; iqnr auQip ibQpih ]"
```
Comment (bhajneet) [inline data/Sri Guru Granth Sahib Ji/1394.json]:
```suggestion
          "SGPC": "iqnr dulIcY bhih; iq nr auQip ibQpih ]"
```

Comment (Sarabveer) [reply data/Sri Guru Granth Sahib Ji/1394.json]: This is wrong according to SGPC Shabadarth, we aren't using Dr. Thind's PDF as proof.

**Comment by Sarabveer** [comment]
@bhajneet for 8bede08526cf7fafffc0b5390e774df09697a3e8, a simple replace was done.

Do we need a image proof for every one?

File: data/Sri Guru Granth Sahib Ji/1394.json (hunk 8 above)
Comment (bhajneet) [reply data/Sri Guru Granth Sahib Ji/1394.json]: This suggestion comes from the image referenced in #1565

**Comment by Harjot1Singh** [comment]
This is marked as WIP, is it?

**Comment by Sarabveer** [comment]
@Harjot1Singh yes, its still WIP

**Comment by Sarabveer** [comment]
Nevermind, merge this in. I will make a new PR for pad shed.

**Pull request merged**